components/authz-service/README.md

```diff
@@ -643,6 +644,7 @@ To add a new default policy, the following is needed:
 
 ### Policy API
 
+<!-- TODO: Can we update it with Open API Doc? -->
```

Comment:
The OpenAPI Policy docs are not yet merged: #2689

Reply:
Okay, then we can waiting until it get merge?